src/main/java/com/guysfromusa/carsgame/v1/MapsResource.java

```diff
@@ -0,0 +1,69 @@
+package com.guysfromusa.carsgame.v1;
+
+import com.guysfromusa.carsgame.entities.MapEntity;
+import com.guysfromusa.carsgame.services.MapService;
+import com.guysfromusa.carsgame.v1.converters.MapConverter;
+import com.guysfromusa.carsgame.v1.model.Map;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.inject.Inject;
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang3.Validate.notNull;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+/**
+ * Created by Sebastian Mikucki, 25.02.18
+ */
+@RestController
+@RequestMapping(value = "/v1/maps", produces = APPLICATION_JSON_UTF8_VALUE)
+public class MapsResource {
+
+    private final MapService mapService;
+
+    private final MapConverter mapConverter;
```

Comment:
should converters be beans? I wouldnt like to mock them, hopefully this wont be needed